lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -43,6 +44,11 @@ describe('ReportRenderer', () => {
         require('../../../../report/html/renderer/performance-category-renderer.js');
     }
     global.PerformanceCategoryRenderer = PerformanceCategoryRenderer;
+    if (!PwaCategoryRenderer) {
+      PwaCategoryRenderer =
```

Comment:
I'm so confused why we needed to do this for `PerformanceCategoryRenderer` did you run into something or was it just straight copy?

Reply:
ah, so the issue is that we require `PerformanceCategoryRenderer` (same with `PwaCategoryRenderer`), but it extends `CategoryRenderer`, assuming `CategoryRenderer` is a global (as it will be when viewing the report in the browser). So if it's required at the beginning of the test file, it immediately errors since the class declaration is run immediately (other things, like accessing the global `URL`, aren't run until a method is actually called).

So I think the solution is either something like this, or we wrap the class declaration in something else (which seems worse). Maybe we can come up with a cleaner version of this dance?

Reply:
well I meant more why there's this weird `if` and it gets stored in a local instead of getting attached straight to global :) but gotcha 👍

Reply:
> well I meant more why there's this weird if and it gets stored in a local instead of getting attached straight to global

ah, I see now. Yeah. I'll fix in the next PR when I add more tests